src/main/java/javaslang/collection/Seq.java

```diff
@@ -540,6 +540,14 @@ private boolean checkPrefix(Traversable<T> t, Traversable<T> prefix) {
     @Override
     Seq<T> remove(T element);
 
+    /**
+     * Removes the element at the specified position in this sequence. Shifts any subsequent elements to the left
+     * (subtracts one from their indices).
+     * @param indx position of element to remove
+     * @return a sequence containing all elements of this without the element at the specified position.
+     */
+    Seq<T> removeIndx(int indx);
```

Comment:
Let's name it `remove(int)` to align it with `set(int, T)`. Will this cause ambiguities when we have a `Seq<Integer>`? Then we should name it `removeAt()` to align it with `splitAt()`.